### PR TITLE
1.0.32 Vulkan Header Updates

### PR DESCRIPTION
**Pull request by mark-lunarg**
For your viewing pleasure.

### CODE REVIEW

File: layers/parameter_validation.cpp [hunk 1]
```diff
@@ -3967,15 +3967,21 @@ VKAPI_ATTR void VKAPI_CALL CmdBindPipeline(VkCommandBuffer commandBuffer, VkPipe
...
-    bool skip_call = false;
+    bool skip = false;
     layer_data *my_data = get_my_data_ptr(get_dispatch_key(commandBuffer), layer_data_map);
     assert(my_data != NULL);
 
-    skip_call |= parameter_validation_vkCmdSetViewport(my_data->report_data, firstViewport, viewportCount, pViewports);
+    skip |= preCmdSetViewport(my_data->report_data, viewportCount, pViewports);
 
-    if (!skip_call) {
+    if (!skip) {
```
Comment (courtney-g) [inline layers/parameter_validation.cpp]: These changes should be on a separate commit.

**Comment by mark-lunarg** [comment]
Split the commit and pushed to Master.  Thanks Courtney.
